README.md

```diff
@@ -28,3 +29,41 @@ that is actually comma-delimited, pass ```:col_sep => ','``` as an option when e
 ```ruby
   SpreadsheetExporter::CSV.from_spreadsheet([["First Name", "Last Name"], ["Bob", "Hoskins"], ["Roger", "Rabbit"]])
 ```
+
+### XLSX with Pick Lists
+
+```ruby
+# data sources are written to a `data` worksheet and may be referenced by
+# multiple rows
+data_sources = {
+  "food_types" => %w[Polenta Paella Papaya],
+  "countries" => %w[Canada Türkiye],
+  "cities" => {"Canada"=>["Sxwōxwiyám", "Toronto"], "Türkiye"=>["Eskişehir", "İzmir", "İstanbul"]}
+}
+
+validations = {
+  "favourite_food" => SpreadsheetExporter::ColumnValidation.new(
```

Comment:
This is my main question about code style.  It's nice to have these structs for safety, etc... but they're kind of ugly.  Would this be better as 

```ruby
validations = {
  "favourite_food" => { data_source: "food_types" }
}
```

?